azure-gaming-pr/reference-architectures/cognitive.md

```diff
@@ -20,10 +20,13 @@ Cognitive Services can be used to improve a variety of gaming scenarios, both in
 * [Text Translation](./cognitive-text-translation.md)
   * Translate text between languages so players across the world can communicate with each other in their native language.
 * [Text to Speech](./cognitive-text-to-speech.md)
-  * Help being everyone into the conversation by translating text to audio.
+  * Help bring everyone into the conversation by synthesizing text to audio.  
+  * Game dialogue prototyping to shorten the amount of time and money spent in product to get the game to market sooner, you can rapidly swap lines of dialog and listen to variations in real-time to ensure accuracy. 
```

Comment:
Consider "Facilitate game dialogue prototyping to shorten the amount of time and money spent in the product by getting game to market sooner. You can rapidly swap lines in a dialog and listen to variations in real-time to ensure accuracy."

Reply:
I've changed the wording to avoid confusion. Thanks.